python/ray/data/datasource/datasource.py

```diff
@@ -27,11 +28,11 @@ class Datasource:
 
     .. note::
         Datasource instances must be serializable, since
-        :meth:`~ray.data.Datasource.create_reader` and
-        :meth:`~ray.data.Datasource.write` are called in remote tasks.
+        :meth:`~ray.data.Datasource.write` is called in remote tasks.
 
     """  # noqa: E501
 
+    @Deprecated(message="TODO")
```

Comment:
Do these need to be filled out?

Reply:
Ah, we don't need them. I've added the `warnings.warn` message, but removed the optional `@Deprecated` message